package.json

```diff
@@ -86,6 +86,8 @@
     "@kbn/pm": "link:packages/kbn-pm",
     "@kbn/test-subj-selector": "link:packages/kbn-test-subj-selector",
     "@kbn/ui-framework": "link:packages/kbn-ui-framework",
+    "@types/lodash": "^4.14.109",
```

Comment:
__nit:__ we still use `lodash: 3.10.1` so there is a chance that it may not be compatible with types from v4, probably `https://www.npmjs.com/package/@types/lodash/v/3.10.2` will be a safer bet.